app/views/placements/schools/placements/preview.html.erb

```diff
@@ -8,7 +8,7 @@
 
 <div class="govuk-width-container">
   <%= render(GovukComponent::NotificationBannerComponent.new(title_text: t(".important"))) do %>
-    <h3 class="govuk-heading-m"><%= t(".this_is_a_preview") %></h3>
+    <h1 class="govuk-heading-m"><%= t(".this_is_a_preview") %></h1>
   <% end %>
```

Comment:
This results in two `<h1>` tags being present on the page.

But having looked into it, I'm not convinced a `<h3>` is the right tag to use either. In fact, I don't think it should be a heading at all.

### What the GOV.UK Design System does

Documentation for the [notification banner](https://design-system.service.gov.uk/components/notification-banner/) component uses a paragraph tag with class name `govuk-notification-banner__heading`:

```html
<p class="govuk-notification-banner__heading">
  You have 7 days left to send your application.
  <a class="govuk-notification-banner__link" href="#">View application</a>.
</p>
```

### GOV.UK Components gem

The components gem can handle [notification banners with headings](https://govuk-components.netlify.app/components/notification-banner/#notification-banner-with-heading) directly:

```erb
<%= 
render(GovukComponent::NotificationBannerComponent.new(title_text: t(".important"))) do |nb|
  nb.with_heading(text: t(".this_is_a_preview"))
end
%>
```

### What to do in this PR

For this PR, would you mind switching over to use `nb.with_heading` as per the gem documentation?

I'll follow-up with a separate card to catch any other uses of `GovukComponent::NotificationBannerComponent` and make sure we're using `nb.with_heading` rather than rolling our own headings as `<h3>` tags.